README.md

```diff
@@ -1 +1,2 @@
-# Fall-API-Homework
\ No newline at end of file
+# Fall-API-Homework
+The first piece of homework!!!
```

Comment:
@MayaBliffeld Great job! Now I want you to make another commit. You can change this text, add an email, add an image. What it is that you want :). We're basically testing how multiple commits work on Pull Requests.